code/software/libs/src/start_serial/init.S

```diff
@@ -11,8 +11,8 @@
 ; This is the initialization code. The loader jumps into this
 ; code after the "kernel" is received via serial.
 ;
-; The first section is linked to be run at $40000, where the
-; loader loads the code. This section copies the rest of the
+; The first section is position independent, but will be loaded

```

Comment:
Hmm, is this still correct? It was changed to $40000 in the 1.2 firmware, and still looks to be set to that (see `code/firmware/rosco_m68k_v1.2/include/system.h`).

Reply:
(Sorry, commented on wrong line, specifically referring to the $28000 on the next line).

Reply:
I believe 0x28000 is the load address if that is what you mean.  I did testing and that was the "org" on init.S before I removed it.  I will triple check. :slightly_smiling_face:

Reply:
Okay, checked.  Thanks for mentioning this as now I see it comes from system.h KERNEL_LOAD_ADDRESS.  Since the code is position independent, it doesn't affect run-time, but it does affect assert (which was extra credit for myself - since I hit it and had to "debug").  So my assert is correct for r1.1, but "conservative" for r1.2.  I will make it match r1.2 (or maybe I can add a revision symbol option...but minor).

Reply:
Hmm, in my defense, the "org" was never changed to 0x40000 I don't think (and this was confusing me WRT 1.2):
https://github.com/rosco-m68k/rosco_m68k/blame/a22b394f441e5f30ea937ef832312a9c1bce3e8d/code/software/libs/src/start_serial/init.S#L29

Reply:
Haha yes, as discussed in Discord I take full responsibility :)

Reply:
All cleaned up, I believe.  I re-wrote some of the comments to better reflect how things are now too.